scipy/spatial/transform/rotation.pyx

```diff
@@ -789,6 +789,8 @@ cdef class Rotation:
             A single vector or a stack of vectors, where `rot_vec[i]` gives
             the ith rotation vector.
         degrees : bool, optional
+            *New Feature:* Added in v1.7.0
+        
             If True, then the given magnitudes are assumed to be in degrees.
             Default is False.
 
```

Comment:
```suggestion
        degrees : bool, optional
            *New Feature:* Added in v1.7.0
        
            If True, then the given magnitudes are assumed to be in degrees.
            Default is False.

            .. versionadded:: 1.7.0

```

Reply:
Cool! I didn't know about this sphinx tag; very handy. Learn something new every day :)